src/core/transactionmanager.js

```diff
@@ -539,6 +540,10 @@ TransactionManager.prototype._prepareRequest = function(tx) {
     submitRequest.txJson(tx.tx_json);
   }
 
+  if (!_.isEmpty(tx.signers)) {
+    submitRequest.message.Signers = tx.signers;
```

Comment:
Yeah, I eventually got `submit` to work by working backward from a correct `tx_blob`... there was an issue with Transaction.complete() / canonical signing flag. Purely ripple-lib API issue